### PR TITLE
UWP uses correct LoadLibrary

### PR DESCRIPTION
**Pull request by ghost**
https://devtopia.esri.com/runtimecore/c_api/issues/9700

UWP applications require a `LoadPackagedLibrary` to be used in place of `LoadLibrary`

These changes use the correct function on UWP, while leaving the Windows Desktop build the same

-----

- [x] Review
- [x] Testing
 - https://devtopia.esri.com/runtimecore/runtimecore/pull/851#issuecomment-1097207
 - http://runtime-test.esri.com:8080/job/c_api-vtest-interface/6877/downstreambuildview/

### CODE REVIEW

File: src/glad.c [hunk 1]
```diff
@@ -771,7 +771,12 @@ PFNWGLGETPROCADDRESSPROC_PRIVATE gladGetProcAddressPtr;
 
 static
 int open_gl(void) {
+#if defined(RTC_WINDOWS_UNIVERSAL)
```
Comment (chri7325) [inline src/glad.c:774]: Would this be better to use a compiler defined ifdef so it could be pushed upstream?

Comment (chri7325) [reply src/glad.c:774]: @dotMorten do you happen to know of any?

Comment (ghost) [reply src/glad.c:774]: This change is in code generated by glad, so it won't be possible to push upstream. I've made an issue for this in the upstream repo: https://github.com/Dav1dde/glad/issues/141

**Review by chri7325 — APPROVED** [review]
Ah okay

**Pull request merged**